contentmgr/pinning.go

```diff
@@ -428,6 +426,8 @@ func (cm *ContentManager) handlePinningComplete(ctx context.Context, handle stri
 		}).Error; err != nil {
 			return xerrors.Errorf("failed to update content in database: %w", err)
 		}
+
+		delete(cm.ZonesConsolidating, cont.ID)
```

Comment:
@neelvirdy add a comment above this - `if the content is a consolidated aggregate, it means all its children have been moved and the content pinned, so remove consolidation flag`.

Good catch!

Reply:
Also, let us stop consolidation altogether where we can avoid it - it's prone to bugs, and it is an expensive process. 

We can discuss this later.